prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -62,7 +69,7 @@ public class InstallCommand extends AbstractInstallCommand {
 
     @CommandLine.Option(
             names = CliConstants.DIR,
-            required = true,
+            required = false,
```

Comment:
the `dir` parameter is still required if `list-profiles` is not present. You need to verify it's set in code and print correct error message, otherwise NPE is thrown.